src/components/FullStatistics.js

```diff
@@ -7,6 +7,16 @@ class FullStatistics extends Component {
   componentDidMount() {
     this.props.fetchAlarms();
   }
+  getDatetimeString(utcRep) {
+    if (utcRep === null) {
+      return null;
+    } else {
+      // Via: https://stackoverflow.com/questions/6525538/convert-utc-date-time-to-local-date-time
+      const d = new Date(utcRep);
+      const tzNonAwareRep = d.toLocaleString();
+      return new Date(tzNonAwareRep + " UTC").toLocaleString();
```

Comment:
Volgens mij klopt dit niet, als utcRep een UTC timestamp is, is new Date(utcRep) al lokaal.

Reply:
fixed